api/managementpb/postgresql.proto

```diff
@@ -78,6 +78,12 @@ message AddPostgreSQLRequest {
   MetricsMode metrics_mode = 21;
   // List of collector names to disable in this exporter.
   repeated string disable_collectors = 22;
+  // TLS CA certificate.
```

Comment:
I think same fields should be also in inventory for postgresql.
https://github.com/percona/pmm/blob/PMM-2.0/api/inventorypb/agents.proto#L754:L754

Reply:
Same fields should be presented for actions and connection check.

Reply:
> Same fields should be presented for actions and connection check.

Where?